frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -734,7 +737,7 @@ const HostDetailsPage = ({
   }, []);
 
   const onCancelAppInstallDetailsModal = useCallback(() => {
```

Comment:
Love using VPP, let's rename anything with App to VPP, including `onCancellAppInstallDetailsModal`